crates/stages/src/stages/merkle.rs

```diff
@@ -0,0 +1,64 @@
+use crate::{
```

Comment:
Probably can leave this stage for separate PR given its a noop right now?

Reply:
It is there for strange logic of execution/unwind, I am afraid it would get lost if removed and it is easier to have it right here.